src/main/java/org/folio/helper/PurchaseOrderHelper.java

```diff
@@ -302,10 +302,10 @@ public Future<Void> updateOrder(CompositePurchaseOrder compPO, boolean deleteHol
                 compPO.setCompositePoLines(clonedPoFromStorage.getCompositePoLines());
               }
               compPO.getCompositePoLines().forEach(poLine -> PoLineCommonUtil.updateLocationsQuantity(poLine.getLocations()));
-              return openCompositeOrderFlowValidator.checkLocationsAndPiecesConsistency(compPO.getCompositePoLines(), requestContext);
-            } else {
-              return Future.succeededFuture();
+              return openCompositeOrderFlowValidator.checkLocationsAndPiecesConsistency(compPO.getCompositePoLines(), requestContext)
```

Comment:
I see that this method checkLocationsAndPiecesConsistency used twice, also in OpenCompositeOrderFlowValidator.
Can we move these 2 validations checkLocationsAndPiecesConsistency and checkFundLocationRestrictions to single place: method validate in OpenCompositeOrderFlowValidator

Reply:
Created a ticket: https://issues.folio.org/browse/MODORDERS-981